spec/indexers/concerns/newspaper_works/indexes_relationships_spec.rb

```diff
@@ -48,16 +45,22 @@
   describe '#index_pages' do
     let(:article) { @page_for_indexrel.articles.first }
     let(:article_indexer) { NewspaperArticleIndexer.new(article) }
-    let(:solr_doc) { {} }
     before { page_indexer.index_pages(article, solr_doc) }
     it 'sets the issue fields correctly' do
       expect(solr_doc['page_ids_ssim'].first).to eq(@page_for_indexrel.id)
       expect(solr_doc['page_titles_ssim'].first).to eq('Page 1')
     end
   end
 
+  describe '#index_siblings' do
+    before { page_indexer.index_siblings(@page_for_indexrel, solr_doc) }
+    it 'sets the prev/next fields correctly' do
+      expect(solr_doc['is_preceding_page_of_ssi']).not_to be_falsey
+      expect(solr_doc['is_following_page_of_ssi']).to be_nil
+    end
```

Comment:
Should we add test expectations particular to first/last page in sequence?

Reply:
Yeah, I could add this. Would just need to add some additional logic to `model_fixtures` to return an array of `NewspaperPage` objects if the proper argument is passed, something like:
```ruby
return [page1, page2] if target_type == :newspaper_pages
```

Reply:
Updated spec to test both the first and last page in the sequence.